src/PerformanceTests/Utils/AssemblyScanner.cs

```diff
@@ -14,7 +14,15 @@ public static IEnumerable<Assembly> GetAssemblies()
 
         foreach (var path in Directory.EnumerateFiles(Environment.CurrentDirectory, "*.dll"))
         {
-            Assembly.LoadFile(path);
+            var assemblyName = Path.GetFileNameWithoutExtension(path);
+            try
+            {
+                Assembly.LoadFrom(path);
```

Comment:
Should we remove the try catch and just let it bomb here?

Reply:
I had Load fail when trying to load a runtime specific dll for the sql client package. As this assembly isn't in the test directory anymore, we should be good to remove the try catch